sw/device/silicon_creator/manuf/lib/individualize.c

```diff
@@ -33,130 +33,120 @@ typedef struct hw_cfg0_settings {
    * instance.
    */
   multi_bit_bool_t en_csrng_sw_app_read;
-  /**
-   * This input efuse is used to enable access to the ENTROPY_DATA register
-   * directly.
-   */
-  multi_bit_bool_t en_entropy_src_fw_read;
-  /**
-   * This input efuse is used to enable access to the firmware override FIFO and
-   * other related functions.
-   */
-  multi_bit_bool_t en_entropy_src_fw_over;
-} hw_cfg0_settings_t;
+} hw_cfg1_settings_t;
 
 // Changing any of the following values may result in unexpected device
 // behavior.
 //
 // - en_sram_ifetch: required to enable SRAM code execution during
 //   manufacturing.
 // - en_csrng_sw_app_read: required to be able to extract output from CSRNG.
-// - en_entropy_src_fw_read and en_entropy_src_fw_over: Required to implement
-//   entropy_src conditioner KAT.
-const hw_cfg0_settings_t kHwCfg0Settings = {
+const hw_cfg1_settings_t kHwCfg1Settings = {
     .en_sram_ifetch = kMultiBitBool8True,
-    .en_csrng_sw_app_read = kMultiBitBool8True,
-    .en_entropy_src_fw_read = kMultiBitBool8True,
-    .en_entropy_src_fw_over = kMultiBitBool8True,
-};
+    .en_csrng_sw_app_read = kMultiBitBool8True};
 
 /**
- * Configures digital logic settings in the HW_CFG0 partition.
+ * Configures digital logic settings in the HW_CFG1 partition.
  *
  * @param otp OTP controller instance.
  * @return The result of the operation.
  */
 OT_WARN_UNUSED_RESULT
-static status_t hw_cfg0_enable_knobs_set(const dif_otp_ctrl_t *otp_ctrl) {
+static status_t hw_cfg1_enable_knobs_set(const dif_otp_ctrl_t *otp_ctrl) {
   uint32_t val =
-      bitfield_field32_write(0, kSramFetch, kHwCfg0Settings.en_sram_ifetch);
+      bitfield_field32_write(0, kSramFetch, kHwCfg1Settings.en_sram_ifetch);
   val = bitfield_field32_write(val, kCsrngAppRead,
-                               kHwCfg0Settings.en_csrng_sw_app_read);
-  val = bitfield_field32_write(val, kEntropySrcFwRd,
-                               kHwCfg0Settings.en_entropy_src_fw_read);
-  val = bitfield_field32_write(val, kEntropySrcFwOvr,
-                               kHwCfg0Settings.en_entropy_src_fw_over);
```

Comment:
Just checking: Are these fields gone now (not moved)? Was that meant to be part of this change?

...assuming I'm not just having trouble seeing them, haha